### ISSUE TITLE
Weird artifact when zooming-in and zooming-out

### ISSUE THREAD

**Issue by Loreno10** [elementary/photos#598]
There is a weird effect anytime I try to make an image bigger, and then smaller.
Example:
1. My image looks like this:
![image](https://user-images.githubusercontent.com/10273406/106725149-557e2000-6609-11eb-8244-adad0136f1b2.png)

2. Then, I zoom-in a bit:
![image](https://user-images.githubusercontent.com/10273406/106725209-6595ff80-6609-11eb-9ec7-5fcb4660853c.png)

3. Next, I zoom-out, and I get this:
![image](https://user-images.githubusercontent.com/10273406/106725260-78103900-6609-11eb-858b-844f81848be3.png)

Basically, the background "buffer" still contains the image instead of the grey color that it should have.
It happens with every image.

**Comment by Loreno10** [comment]
I'm on eOS 5.1.7 by the way

**Comment by cassidyjames** [comment]
@Loreno10 what stylesheet are you using? It looks like you might be using an old snapshot of the unreleased stylesheet for elementary OS 6, which is not supported on elementary OS 5.

**Comment by Loreno10** [comment]
Yeah, you;re right. It happens only when I'm using the eOS6 stylesheet. On the default one, this issue does not happen.
